xbmc/games/addons/GameClientSubsystem.cpp

```diff
@@ -32,6 +33,7 @@ GameClientSubsystems CGameClientSubsystem::CreateSubsystems(CGameClient& gameCli
 {
   GameClientSubsystems subsystems = {};
 
+  subsystems.Cheevos.reset(new CGameClientCheevos(gameClient, gameStruct));
```

Comment:
```suggestion
  subsystems.Cheevos = std::make_unique<CGameClientCheevos>(gameClient, gameStruct);
```

Reply:
Should I update the other 3 subsystems which do the same thing with reset and new?

Reply:
@NikosSiak Let's save that for a later PR